rmw_fastrtps_shared_cpp/src/rmw_logging.cpp

```diff
@@ -49,7 +49,7 @@ __rmw_set_log_severity(rmw_log_severity_t severity)
       return RMW_RET_ERROR;
   }
 
-  eprosima::fastrtps::Log::SetVerbosity(log_kind);
+  eprosima::fastdds::dds::Log::SetVerbosity(log_kind);
```

Comment:
```suggestion
  Log::SetVerbosity(log_kind);
```

Previously it has been stated `using eprosima::fastdds::dds::Log`